### DISCUSSION TITLE
New to Github

### DISCUSSION THREAD

**Discussion by irrationalwork** [community/community#43091] [New to GitHub] [answered]
how to improve my github knowledge?? I'm not able to start how this thing workss!!

**Comment by diva-21** [comment, marked as answer]
Have a good and personalized README.md
Visit [Eddie Jaoude course ](https://www.eddiejaoude.io/course-github-profile-landing) for README customization.


On youtube, follow Kunal Kushwaha , Eddie and other Open Source guys at Twitter.
Most of your answers are on Twitter daily
Join communities, start looking issues where ever your understanding is fitting

Try to get badges
Use emojis to clarify tone. Compare “Looks good :)” or “Looks good! 👍 ” with “It’s fine.”
Commit small changes
Good commit messages describing what the commit is for


Do it daily with consistency , and you will love the process.

**Reply by irrationalwork** [reply]
Thanks @diva-21, i will try to start my 2023 with open source

**Reply by diva-21** [reply]
And also i forgot to tell, try to keep Pull requests(prs) in the known domains of urs.
After certain amount of time, try to understand Devops (like Linux,docker,kubernetes,etc) .

All the best for your new Journey

**Reply by irrationalwork** [reply]
Sure  Its a great talk.

**Reply by diva-21** [reply]
You can visit my profile at Github for more info 😎
